Day-I-Social-Card/social-card/src/App.js

```diff
@@ -1,23 +1,13 @@
 import React from 'react';
 import './App.css';
+import CardContainer from '../src/components/CardComponents/CardContainer';
 
 const App = () => {
+  
   return (
-    <div>
-      <h3>Welcome to React Social Card!</h3>
-      <p>
-        Begin by exploring the `components` directory. You'll notice we have a
-        few files that we've already included in there to get you started right
-        away building components. You'll need to make sure you include your
-        components that you build in this file to watch your app come to life
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
+   <div>
+   <CardContainer/>
```

Comment:
When returning only 1 HTML element, you don't need the div parent container. The purpose of wrapping components/HTML elements with the parent div is so you're only returning 1 element, so if you're only returning 1 element to begin with you don't need that parent wrapper.